client/e2e/pages/onboarding/home/page.spec.ts

```diff
@@ -1,18 +1,73 @@
-// 🧪 Suite to test the Home page `http://localhost:3000/home`
-// 🔍 Asserts the user can login, logout, and login
+// 🧪 Suite to test the onboarding\Home page `http://localhost:3000/home`
```

Comment:
Why is the file called onboarding?

Reply:
So to reflect `client/app/(onboarding)/home/page.tsx`